### DISCUSSION TITLE
building against the eclipse ide platform with tycho 1.5.1!

### DISCUSSION THREAD

**Discussion by nadhemB** [eclipse-tycho/tycho#346] [General]
I want to add the plugins in my eclipse ide folder to my target platform. 
I thought about creating a bundle from the existing jars but some of the plugins which re not packed as jars will not be included!
any way around?

**Comment by laeubi** [comment]
You can reference your _installation_ with a `file:<path to your eclipse>` in a repository url. But be aware that this is not portable at all.

**Reply by nadhemB** [reply]
is it possible to create an update site from all the plugins in my eclipse folder and then include that site in my target-platform?

**Reply by laeubi** [reply]
No but you can upgrade your tycho version and use a Directory based Target-Location instead.

**Reply by nadhemB** [reply]
thanks alot sir!
